src/pages/test-report/components/competency/competency.ts

```diff
@@ -116,7 +108,7 @@ export class CompetencyComponent {
       hasDangerousFault$.pipe(map(toggle => this.hasDangerousFault = toggle)),
     );
 
-    this.subscription = merged$.subscribe();
+    this.subscription = merged$.subscribe(() => this.canButtonRipple());
```

Comment:
I feel there might be a better way to do this?

Reply:
If you just want to keep your subscribe a clean parameter-less call I think you could pipe off the merged$...
```
    const merged$ = merge(
      drivingFaultCount$.pipe(map(count => this.faultCount = count)),
      isRemoveFaultMode$.pipe(map(toggle => this.isRemoveFaultMode = toggle)),
      isSeriousMode$.pipe(map(toggle => this.isSeriousMode = toggle)),
      hasSeriousFault$.pipe(map(toggle => this.hasSeriousFault = toggle)),
      isDangerousMode$.pipe(map(toggle => this.isDangerousMode = toggle)),
      hasDangerousFault$.pipe(map(toggle => this.hasDangerousFault = toggle)),
    )
    .pipe(tap(this.canButtonRipple()));

    this.subscription = merged$.subscribe();
```

Reply:
Great, that works a treat. Cleaner and more like what I had in mind.